src/AspNetCoreModuleV2/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -1138,11 +1139,16 @@ FORWARDING_HANDLER::CreateWinHttpRequest(
         goto Finished;
     }
 
+    if (!pServerProcess->IsDebuggerAttached())
```

Comment:
Port this to V1?

Reply:
yes. I also need port couple of other changes to V1. will create another PR for porting.